infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -159,6 +162,15 @@ module "worklytics-psoxy-connection" {
 # BEGIN LONG ACCESS AUTH CONNECTORS
 locals {
   long_access_parameters = { for entry in module.worklytics_connector_specs.enabled_oauth_secrets_to_create : "${entry.connector_name}.${entry.secret_name}" => entry }
+  env_vars_for_locker = distinct(flatten([
+  for k, v in module.worklytics_connector_specs.enabled_oauth_long_access_connectors : [
+  for env_var in v.environment_variables : {
+    connector_name = k
+    env_var_name    = "OAUTH_REFRESH_TOKEN"
+  } if try(v.environment_variables.USE_SHARED_TOKEN, null) != null
+  ] if try(v.environment_variables, null) != null
+  ]))
+  env_vars_for_locker_for_locker_parameters = { for entry in local.env_vars_for_locker : "${entry.connector_name}.${entry.env_var_name}" => entry }
```

Comment:
Instead of creating a *fake* secret through specs, I've move it to the combination of new *gcp-instance-secret-locker* module and to the variable created here. This is something specific for GCP due the way we work with labels for sharing thing.

Reply:
why `for_locker` twice?